database/migrations/default/1731003209538_rename_is_coa_roadway/up.sql

```diff
@@ -1,7 +1,7 @@
 alter table crashes rename column is_non_coa_roadway to is_coa_roadway;
 alter table crashes alter column is_coa_roadway set default false;
 
-comment on column public.crashes.is_coa_roadway is 'If the crash location occured within the City of Austin, not on a private drive, and not intersecting with the non_coa_roadways layer. Set via trigger.';
+comment on column public.crashes.is_coa_roadway is 'If the crash location occurred within the City of Austin, not on a private drive, and not intersecting with the non_coa_roadways layer. Set via trigger.';
```

Comment:
i know this is harmless, but it feels odd making this change since it won't be applied to the db. sadly, we'd need to create a new migration to fix my dumb typo 🫠

Reply:
yea, it is strange changing it now, but the rational in making the change here is not so much to fix it in the DB, but more to keep my code editor from flagging the file as having an issue. it's not often that i do have this file open though, so i'll revert this so that we have a shot at finding it again down the road. thanks!